src/files.js

```diff
@@ -245,6 +245,26 @@ export function getFilePath (cozy, file = {}, folder) {
   return `${folderPath}${file.name}`
 }
 
+export function getShareLink (cozy, id) {
+  if (!id) {
+    return Promise.reject(Error('An id should be provided to create a share link'))
```

Comment:
Oh, no, it's a mistake.
I rewrite from (copy/paste from https://github.com/cozy/cozy-client-js/pull/160/files#diff-ecdbeb0b77447177e8165fbaf62b5a09R238)

```js
throw Error('An id should be provided to create a share link')
```

to this code.